xml_inscriptions/transcribed/UT.Provo.BYU.HBLL.G.2011.07.01.xml

```diff
@@ -0,0 +1,171 @@
+                        </origin>
+                        <provenance type="transferred" subtype="acquired" when-custom="1983-1986"> <p>From the <orgName>Royal Athena Gallery in New York</orgName> by a <persName role="acquier">Mr.
+                                    David H. Swingler</persName> of <orgName>Oxnard,
+                                    California</orgName> between <date notBefore="1982"
+                                    notAfter="1986">1983-1985</date></p></provenance>
+                        <provenance type="transferred" subtype="donated" when="2006"><p>y <persName role="donor">Mr. Swingler</persName> to the <orgName>Harold B. Lee Library at Brigham
+                                    Young University, Utah</orgName> in <date notBefore="2005"
+                                    notAfter="2007">2006</date></p></provenance>
+
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#"/>
+                        <biblScope unit="page">ZPE_177_Blumell</biblScope>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2020-09-16" who="Itamar Levin">Initial Encoding</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url=""/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+                
+                <lb n="1"/>θάρσι <persName key="Heraklinos">Ἡρα
+                    <lb n="2" break="no"/>κλιανέ</persName>, οὐ
+                <lb n="3" break="no"/>δὶς ἀθάνα
+                <lb n="4" break="no"/>τος.
+                
+                
+            </ab></div>
+
+            <div type="commentary">
+            <p><lb n="1"/>Line 1: Ι. Θάρσει. Line 2-3: I. οὐδεὶς <lb n="2"/> The item and
```

Comment:
I'd say that these are less commentary and more apparatus, and should be in that `<div>` instead. You may also want to mark up the text itself accordingly.